
<!DOCTYPE html>
<html>
<head>
<!--
ExtentReports Library 2.40.1 | http://relevantcodes.com/extentreports-for-selenium/ | https://github.com/anshooarora/
Copyright (c) 2015, Anshoo Arora (Relevant Codes) | Copyrights licensed under the New BSD License | http://opensource.org/licenses/BSD-3-Clause
Documentation: http://extentreports.relevantcodes.com 
-->

<extent id='4c67209b-cac3-46b0-80e3-544ccf75b9bf' />

<meta charset='UTF-8' /> 
<meta name='description' content='ExtentReports (by Anshoo Arora) is a reporting library for automation testing for .NET and Java. It creates detailed and beautiful HTML reports for modern browsers. ExtentReports shows test and step summary along with dashboards, system and environment details for quick analysis of your tests.' />
<meta name='robots' content='noodp, noydir' />
<meta name='viewport' content='width=device-width, initial-scale=1' />

<title>
Sample Automation Reports
</title>


<link href='https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css' type='text/css' rel='stylesheet'>
<link href='https://cdn.rawgit.com/anshooarora/extentreports/a7fadea2f449184064e06465391f689653dea590/cdn/extent.css' type='text/css' rel='stylesheet' />

<style>





</style>
</head>
<body class='extent'>
<!-- nav -->
<nav>
<div class='logo-container'>
<a class='logo-content' href='http://extentreports.relevantcodes.com'>
<span>ExtentReports</span>
</a>
<a class='menu-toggle right' style='display:none;'>
<i class='mdi-navigation-menu'></i>
</a>
</div>
<ul id='slide-out' class='side-nav fixed'>
<li class='analysis waves-effect active'><a href='#!' class='test-view' onclick="_updateCurrentStage(0)"><i class='mdi-action-dashboard'></i>Test Details</a></li>
<li class='analysis waves-effect'><a href='#!' class='exceptions-view' onclick="_updateCurrentStage(2)"><i class='mdi-action-bug-report'></i>Exceptions</a></li>
<li class='analysis waves-effect'><a href='#!' class='dashboard-view'><i class='mdi-action-track-changes'></i></i>Analysis</a></li>
</ul>
<a href='#' data-activates='slide-out' class='button-collapse'><i class='mdi-navigation-apps medium'></i></a>
<span class='report-name'>Automation Report</span> <span class='report-headline'>Taxilla</span>
<ul class='right hide-on-med-and-down nav-right'>
<li class='theme-selector' alt='Click to toggle dark theme. To enable by default, use js configuration $("body").addClass("dark");' title='Click to toggle dark theme. To enable by default, use js configuration $("body").addClass("dark");'>
<i class='mdi-hardware-desktop-windows'></i>
</li>
<li>
<span class='suite-started-time'>2021-07-15 17:00:54</span>
</li>
<li>
<span>v2.40.1</span>
</li>
</ul>
</nav>
<!-- /nav -->

<!-- container -->
<div class='container'>

<!-- dashboard -->
<div id='dashboard-view' class='row'>
<div class='time-totals'>
<div class='col l2 m4 s6'>
<div class='card suite-total-tests'> 
<span class='panel-name'>Total Tests</span> 
<span class='total-tests'> <span class='panel-lead'></span> </span> 
</div> 
</div>
<div class='col l2 m4 s6'>
<div class='card suite-total-steps'> 
<span class='panel-name'>Total Steps</span> 
<span class='total-steps'> <span class='panel-lead'></span> </span> 
</div> 
</div>
<div class='col l2 m4 s12'>
<div class='card suite-total-time-current'> 
<span class='panel-name'>Total Time Taken (Current Run)</span> 
<span class='suite-total-time-current-value panel-lead'>0h 0m 43s+670ms</span> 
</div> 
</div>
<div class='col l2 m4 s12'>
<div class='card suite-total-time-overall'> 
<span class='panel-name'>Total Time Taken (Overall)</span> 
<span class='suite-total-time-overall-value panel-lead'>0h 0m 43s+670ms</span> 
</div> 
</div>
<div class='col l2 m6 s6 suite-start-time'>
<div class='card green-accent'> 
<span class='panel-name'>Start</span> 
<span class='panel-lead suite-started-time'>2021-07-15 17:00:11</span> 
</div> 
</div>
<div class='col l2 m6 s6 suite-end-time'>
<div class='card pink-accent'> 
<span class='panel-name'>End</span> 
<span class='panel-lead suite-ended-time'>2021-07-15 17:00:54</span> 
</div> 
</div>
</div>
<div class='charts'>
<div class='col s12 m6 l4 fh'> 
<div class='card-panel'> 
<div>
<span class='panel-name'>Tests View</span>
</div> 
<div class='panel-setting modal-trigger test-count-setting right'>
<a href='#test-count-setting'><i class='mdi-navigation-more-vert text-md'></i></a>
</div> 
<div class='chart-box'>
<canvas class='text-centered' id='test-analysis'></canvas>
</div> 
<div>
<span class='weight-light'><span class='t-pass-count weight-normal'></span> test(s) passed</span>
</div> 
<div>
<span class='weight-light'><span class='t-fail-count weight-normal'></span> test(s) failed, <span class='t-others-count weight-normal'></span> others</span>
</div> 
</div> 
</div> 
<div class='col s12 m6 l4 fh'> 
<div class='card-panel'> 
<div>
<span class='panel-name'>Steps View</span>
</div> 
<div class='panel-setting modal-trigger step-status-filter right'>
<a href='#step-status-filter'><i class='mdi-navigation-more-vert text-md'></i></a>
</div> 
<div class='chart-box'>
<canvas class='text-centered' id='step-analysis'></canvas>
</div> 
<div>
<span class='weight-light'><span class='s-pass-count weight-normal'></span> step(s) passed </span>
</div> 
<div>
<span class='weight-light'><span class='s-fail-count weight-normal'></span> step(s) failed, <span class='s-others-count weight-normal'></span> others</span>
</div> 
</div> 
</div>
<div class='col s12 m12 l4 fh'> 
<div class='card-panel'> 
<span class='panel-name'>Pass Percentage</span> 
<span class='pass-percentage panel-lead'></span> 
<div class='progress light-blue lighten-3'> 
<div class='determinate light-blue'></div> 
</div> 
</div> 
</div>
</div>
<div class='system-view'>
<div class='col l4 m12 s12'>
<div class='card-panel'>
<span class='label info outline right'>Environment</span>
<table>
<thead>
<tr>
<th>Param</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>OS</td>
<td>Linux</td>
</tr>
<tr>
<td>User Name</td>
<td>aknandigam</td>
</tr>
<tr>
<td>Java Version</td>
<td>1.8.0_292</td>
</tr>
<tr>
<td>Environment</td>
<td>QA</td>
</tr>
<tr>
<td>Host Name</td>
<td>AIPLLTH321</td>
</tr>
<tr>
<td>Selenium Version</td>
<td>2.53.0</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class='category-summary-view'>
<div class='col l4 m6 s12'>
<div class='card-panel'>
<span class='label info outline right'>Categories</span>
<table>
<thead>
<tr>
<th>Name</th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
</div>
</div>
</div>
<!-- /dashboard -->

<!-- tests -->
<div id='test-view' class='row _addedTable'>
<div class='col _addedCell1'>
<div class='contents'>
<div class='card-panel heading'>
<h5>Tests</h5>
</div>
<div class='card-panel filters'>
<div>
<a data-activates='tests-toggle' data-constrainwidth='true' data-beloworigin='true' data-hover='true' href='#' class='dropdown-button button tests-toggle'><i class='mdi-action-subject icon'></i></a>
<ul id='tests-toggle' class='dropdown-content'>
<li class='pass'><a href='#!'>Pass</a></li>
<li class='fail'><a href='#!'>Fail</a></li>
<li class='skip'><a href='#!'>Skip</a></li>
<li class='divider'></li>
<li class='clear'><a href='#!'>Clear Filters</a></li>
</ul>
</div>
<div>
<a id='clear-filters' alt='Clear Filters' title='Clear Filters'><i class='mdi-navigation-close icon'></i></a>
</div>
<div>&nbsp;&middot;&nbsp;</div>
<div>
<a id='enableDashboard' alt='Enable Dashboard' title='Enable Dashboard'><i class='mdi-action-track-changes icon'></i></a>
</div>
<div>
<a id='refreshCharts' alt='Refresh Charts on Filters' title='Refresh Charts on Filters' class='enabled'><i class='mdi-navigation-refresh icon'></i></i></a>
</div>
<div>&nbsp;&middot;</div>
<div class='search' alt='Search Tests' title='Search Tests'>
<div class='input-field left'>
<input id='searchTests' type='text' class='validate' placeholder='Search Tests...'>
</div>
<i class='mdi-action-search icon'></i>
</div>
</div>
<div class='card-panel no-padding-h no-padding-v no-margin-v'>
<div class='wrapper'>
<ul id='test-collection' class='test-collection'>
<li class='collection-item test displayed active fail' extentid='6d04b9c8-17f0-488b-a6b5-f1e83e94b85b'>
<div class='test-head'>
<span class='test-name'>Partner Manegement TestCases <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:11</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 43s+163ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='a21190d6-6adc-4339-a312-903442cc0a78'>
<div class='test-head'>
<span class='test-name'>1)Verifying Partner Management Menuitem is Displaying or not <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:11</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 42s+576ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:11</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:11</td>
<td class='step-details'>{Security_Ans2=test, Username=Honourmobile, Security_Ans1=test, Password=Test@123, Runmode=Y, Browser=Chrome}</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:13</td>
<td class='step-details'>Browser has oppened</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:19</td>
<td class='step-details'>URL has oppened</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:19</td>
<td class='step-details'>Trying to login:Honourmobile,Test@123</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:22</td>
<td class='step-details'>Verifying whether element is enabled/clickable</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:22</td>
<td class='step-details'>Text-box  is enabled</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:23</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_22_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_22_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:23</td>
<td class='step-details'>Verifying whether element is enabled/clickable</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:23</td>
<td class='step-details'>Text-box  is enabled</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:23</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_23_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_23_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:24</td>
<td class='step-details'>Verifying whether element is enabled/clickable</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:24</td>
<td class='step-details'>Element is clickable</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:24</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_24_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_24_IST_2021.png' /></td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:28</td>
<td class='step-details'>First level Login is succesfull</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:29</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_28_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_28_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:30</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_30_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_30_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:30</td>
<td class='step-details'>Trying to enter security answers:test,test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:30</td>
<td class='step-details'>Verifying whether element is enabled/clickable</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:30</td>
<td class='step-details'>Text-box  is enabled</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:30</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_30_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_30_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:31</td>
<td class='step-details'>Verifying whether element is enabled/clickable</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:31</td>
<td class='step-details'>Text-box  is enabled</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:31</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_31_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_31_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:31</td>
<td class='step-details'>Verifying whether element is enabled/clickable</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:31</td>
<td class='step-details'>Element is clickable</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:31</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_31_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_31_IST_2021.png' /></td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:39</td>
<td class='step-details'>Login succesfully</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:39</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_39_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_39_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:47</td>
<td class='step-details'>Checking whether PartnerManagment menu item is displaying or not</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:47</td>
<td class='step-details'>PartnerManagment menu item is displaying</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:48</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_47_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_47_IST_2021.png' /></td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='d25c06c8-561b-4ec1-8a28-598652e06750'>
<div class='test-head'>
<span class='test-name'>2)Verifying whether all the menu items are displaying in Partner details page or not <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:48</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 5s+961ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:48</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:48</td>
<td class='step-details'>{Security_Ans2=test, Username=Honourmobile, Security_Ans1=test, Password=Test@123, Runmode=Y, Browser=Chrome}</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:48</td>
<td class='step-details'>Clicking on PartnerMangement Menu Item</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:49</td>
<td class='step-details'>Verifying whether element is enabled/clickable</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:49</td>
<td class='step-details'>Element is clickable</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:49</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_49_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_49_IST_2021.png' /></td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Verifying Partner details Page</td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>AssociatePartner menu item is displaying</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_53_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_53_IST_2021.png' /></td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>ManagePartner menu item is displaying</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_53_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_53_IST_2021.png' /></td>
</tr>
<tr>
<td class='status pass' title='pass' alt='pass'><i class='mdi-action-check-circle'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Add Partner button are displaying</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Screenshot-> <img class='report-img' data-featherlight='./screenshot/Thu_Jul_15_17_00_53_IST_2021.png' src='./screenshot/Thu_Jul_15_17_00_53_IST_2021.png' /></td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='a72dc87d-305c-45bd-8212-b368d52da740'>
<div class='test-head'>
<span class='test-name'>3)Verifying whether Non Service provider organization is getting displayed under Associate Partners <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:53</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+561ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>{Security_Ans2=test, Username=Honourmobile, Security_Ans1=test, Password=Test@123, Runmode=Y, Browser=Chrome}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='2a0c1b50-557e-4536-a5dd-f4745cb75237'>
<div class='test-head'>
<span class='test-name'>4)Verifying whether we are able to associate organization as partner <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:53</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+479ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>{Security_Ans2=test, Username=Honourmobile, Security_Ans1=test, Password=Test@123, Runmode=Y, Browser=Chrome}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:53</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='9c3a3113-47ad-4690-8d4b-b68291398413'>
<div class='test-head'>
<span class='test-name'>5)Verifying whether proper error message is getting displayed when trying to add already Partner <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+434ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{Security_Ans2=test, Username=Honourmobile, Security_Ans1=test, Password=Test@123, Runmode=Y, Browser=Chrome}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='3791a819-ee4b-4f74-8fee-6edcf69eddf2'>
<div class='test-head'>
<span class='test-name'>6)Verifying whether we are able to reject the partnership before any action is performed on the partnership request from Partner org <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+384ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{Security_Ans2=test, Username=Honourmobile, Security_Ans1=test, Password=Test@123, Runmode=Y, Browser=Chrome}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='720a8683-ad39-493b-ac53-4d02f08afadb'>
<div class='test-head'>
<span class='test-name'>7)Verifying whether WQ is getting generated at partner org for partnership approval and also partnership related record is getting added at Manage Partners Tab <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+336ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='830dc689-56c1-4d46-8723-abba2a3764fc'>
<div class='test-head'>
<span class='test-name'>8)Verifying whether Partner org user is able to reject the partnership approval from WQ,once rejected related record is getting deleted in main org <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+295ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='30bd6215-64a7-4963-8a72-80434aacb985'>
<div class='test-head'>
<span class='test-name'>9)Verifying whether proper error message is getting displayed when we try to access the WQ for which rejection action is already performed <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+250ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='ec1e657d-b134-443a-833d-516f9f74e545'>
<div class='test-head'>
<span class='test-name'>10)Verifying whether Partner org user is able to reject the partnership approval from Manage Partner tab,once rejected related record is getting deleted in main org <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+212ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='eacfa76d-1788-4706-971f-fdb0cedaf29f'>
<div class='test-head'>
<span class='test-name'>11)Verifying whether Partner org user is able to Approve the partnership request from W.Q's,once approved verifying below steps: <br />a)Checking whether succes message is displaying or not <br />b)Status related to the request record is changing to Approved or not<br />c)In Assign Users page,partner org is displaying or not <br />d)Verify whether we are able to assign role for partner org user <br />e)Verify whether partner org user is able to switch to main org <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+178ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='1d65d2ed-2f1b-4cb3-a082-f5e3b2960404'>
<div class='test-head'>
<span class='test-name'>12)Verifying whether proper error message is getting displayed when we try to access the WQ for which Approved action is already performed <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+141ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='887cfdf2-5a1c-4698-9266-264575ca2ddd'>
<div class='test-head'>
<span class='test-name'>13)Verifying below steps after rejecting the approved partnership for main org: <br />a)Checking whether we are able to reject the already approved partnership <br />b)Checking whether relevant record is getting deleted in main org <br />c)Checking whether partner is displaying under assign users drop-down <br />d)Checking whether relevant record is getting deleted at partner org <br />e)Checking whether partner org user is able to switch to main org <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+103ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active fail' extentid='9d475cf3-9698-48bb-aa32-f26d1fc4c703'>
<div class='test-head'>
<span class='test-name'>14)Verifying whether Partner org user is able to Approve the partnership request from Manage Partner tab,once approved verifying below steps: <br />a)Checking whether success message is displayed and status is getting updated to Approved in partner org <br />b)Status related to the request record is changing to Approved in main org <br />c)In Assign Users page,partner org is displaying or not <br />d)Verify whether we are able to assign role for partner org user <br />e)Verify whether partner org user is able to switch to main org <i class='tooltipped mdi-alert-error' data-position='top' data-delay='50' data-tooltip='Close was called before test could end safely using EndTest.'></i></span>
<span class='test-status label right outline capitalize fail'>fail</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+50ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
<tr>
<td class='status fail' title='fail' alt='fail'><i class='mdi-navigation-cancel'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'><pre>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
<li class='collection-item test displayed active skip' extentid='e5c72869-49bf-4f5b-8040-654d66905278'>
<div class='test-head'>
<span class='test-name'>15)Verifying below steps after rejecting the approved partnership for Partner org: <br />a)Checking whether we are able to reject the already approved partnership <br />b)Checking whether relevant record is getting deleted in Partner org <br />c)Checking whether partner is displaying under assign users drop-down <br />d)Checking whether relevant record is getting deleted at partner org <br />e)Checking whether partner org user is able to switch to main org </span>
<span class='test-status label right outline capitalize skip'>skip</span>
<span class='category-assigned hide '></span>
</div>
<div class='test-body'>
<div class='test-info'>
<span title='Test started time' alt='Test started time' class='test-started-time label green lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Test ended time' alt='Test ended time' class='test-ended-time label red lighten-2 text-white'>2021-07-15 17:00:54</span>
<span title='Time taken to finish' alt='Time taken to finish' class='test-time-taken label blue-grey lighten-3 text-white'>0h 0m 0s+2ms</span>
</div>
<div class='test-desc'></div>
<div class='test-attributes'>
</div>
<div class='test-steps'>
<table class='bordered table-results'>
<thead>
<tr>
<th>Status</th>
<th>Timestamp</th>
<th>Details</th>
</tr>
</thead>
<tbody>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Start the test</td>
</tr>
<tr>
<td class='status info' title='info' alt='info'><i class='mdi-action-info-outline'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>{PatnerPassword=Test@2233, Runmode=Y, OrgPassword=Test@123, PatnerUsername=searchorg, OrgSecurity_Ans2=test, PatnerSecurity_Ans2=test, Browser=Chrome, OrgSecurity_Ans1=test, PatnerSecurity_Ans1=test, OrgUsername=Honourmobile}</td>
</tr>
<tr>
<td class='status skip' title='skip' alt='skip'><i class='mdi-content-redo'></i></td>
<td class='timestamp'>17:00:54</td>
<td class='step-details'>Test case is skipped as Runmode is N</td>
</tr>
</tbody>
</table>
<ul class='collapsible node-list' data-collapsible='accordion'>
</ul>
</div>
</div>
</li>
</ul>
</div>
</div>
</div>
</div>
<div id='test-details-wrapper' class='col _addedCell2'>
<div class='contents'>
<div class='card-panel details-view'>
<h5 class='details-name'></h5>
<div class='details-container'>
</div>
</div>
</div>
</div>
</div>
<!-- /tests -->

<!-- categories -->
<!-- /categories -->

<!-- exceptions -->
<div id='exceptions-view' class='row _addedTable hide'>
<div class='col _addedCell1'>
<div class='contents'>
<div class='card-panel heading'>
<h5>Defects</h5>
</div>
<div class='card-panel filters'>
<div class='search' alt='Search tests' title='Search tests'>
<div class='input-field left'>
<input id='searchTests' type='text' class='validate' placeholder='Search tests...'>
</div>
<i class='mdi-action-search icon'></i>
</div>
</div>
<div class='card-panel no-padding-h no-padding-v'>
<div class='wrapper'>
<ul id='exception-collection' class='exception-collection'>
<li class='exception-item displayed'>
<div class='exception-head'>
<span class='exception-name'>com.relevantcodes.extentreports.ExtentTestInterruptedException</span>
</div>
<div class='exception-test-count'>
<span class='fail label'>15</span>
</div>
<div class='exception-body'>
<div class='exception-tests'>
<table class='bordered'>
<thead>
<tr>
<th>RunDate</th>
<th>Test Name</th>
<th>Exception</th>
</tr>
</thead>
<tbody>
<tr class='fail'>
<td>2021-07-15 17:00:11</td>
<td><span class='exception-link linked' extentid='6d04b9c8-17f0-488b-a6b5-f1e83e94b85b'>Partner Manegement TestCases</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:11</td>
<td><span class='exception-link linked' extentid='a21190d6-6adc-4339-a312-903442cc0a78'>1)Verifying Partner Management Menuitem is Displaying or not</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:48</td>
<td><span class='exception-link linked' extentid='d25c06c8-561b-4ec1-8a28-598652e06750'>2)Verifying whether all the menu items are displaying in Partner details page or not</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:53</td>
<td><span class='exception-link linked' extentid='a72dc87d-305c-45bd-8212-b368d52da740'>3)Verifying whether Non Service provider organization is getting displayed under Associate Partners</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:53</td>
<td><span class='exception-link linked' extentid='2a0c1b50-557e-4536-a5dd-f4745cb75237'>4)Verifying whether we are able to associate organization as partner</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='9c3a3113-47ad-4690-8d4b-b68291398413'>5)Verifying whether proper error message is getting displayed when trying to add already Partner</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='3791a819-ee4b-4f74-8fee-6edcf69eddf2'>6)Verifying whether we are able to reject the partnership before any action is performed on the partnership request from Partner org</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='720a8683-ad39-493b-ac53-4d02f08afadb'>7)Verifying whether WQ is getting generated at partner org for partnership approval and also partnership related record is getting added at Manage Partners Tab</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='830dc689-56c1-4d46-8723-abba2a3764fc'>8)Verifying whether Partner org user is able to reject the partnership approval from WQ,once rejected related record is getting deleted in main org</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='30bd6215-64a7-4963-8a72-80434aacb985'>9)Verifying whether proper error message is getting displayed when we try to access the WQ for which rejection action is already performed</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='ec1e657d-b134-443a-833d-516f9f74e545'>10)Verifying whether Partner org user is able to reject the partnership approval from Manage Partner tab,once rejected related record is getting deleted in main org</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='eacfa76d-1788-4706-971f-fdb0cedaf29f'>11)Verifying whether Partner org user is able to Approve the partnership request from W.Q's,once approved verifying below steps: <br />a)Checking whether succes message is displaying or not <br />b)Status related to the request record is changing to Approved or not<br />c)In Assign Users page,partner org is displaying or not <br />d)Verify whether we are able to assign role for partner org user <br />e)Verify whether partner org user is able to switch to main org</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='1d65d2ed-2f1b-4cb3-a082-f5e3b2960404'>12)Verifying whether proper error message is getting displayed when we try to access the WQ for which Approved action is already performed</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='887cfdf2-5a1c-4698-9266-264575ca2ddd'>13)Verifying below steps after rejecting the approved partnership for main org: <br />a)Checking whether we are able to reject the already approved partnership <br />b)Checking whether relevant record is getting deleted in main org <br />c)Checking whether partner is displaying under assign users drop-down <br />d)Checking whether relevant record is getting deleted at partner org <br />e)Checking whether partner org user is able to switch to main org</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tr class='fail'>
<td>2021-07-15 17:00:54</td>
<td><span class='exception-link linked' extentid='9d475cf3-9698-48bb-aa32-f26d1fc4c703'>14)Verifying whether Partner org user is able to Approve the partnership request from Manage Partner tab,once approved verifying below steps: <br />a)Checking whether success message is displayed and status is getting updated to Approved in partner org <br />b)Status related to the request record is changing to Approved in main org <br />c)In Assign Users page,partner org is displaying or not <br />d)Verify whether we are able to assign role for partner org user <br />e)Verify whether partner org user is able to switch to main org</span></td>
<td><pre class='exception-message'>com.relevantcodes.extentreports.ExtentTestInterruptedException: Close was called before test could end safely using EndTest.
at com.relevantcodes.extentreports.Report.terminate(Report.java:401)
at com.relevantcodes.extentreports.ExtentReports.close(ExtentReports.java:702)
at com.selenium.DataDriven.TestCases.Partnermanagment.testb(Partnermanagment.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
at org.testng.TestRunner.privateRun(TestRunner.java:756)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
at org.testng.SuiteRunner.run(SuiteRunner.java:289)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
at org.testng.TestNG.runSuites(TestNG.java:1133)
at org.testng.TestNG.run(TestNG.java:1104)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</pre></td>
</tr>
<tbody>
</table>
</div>
</div> 
</li>
</ul>
</div>
</div>
</div>
</div>
<div id='exception-details-wrapper' class='col _addedCell2'>
<div class='contents'>
<div class='card-panel details-view'>
<h5 class='exception-name'></h5>
<div class='exception-container'>
</div>
</div>
</div>
</div>
</div>
<!-- /exceptions -->

<!-- testrunner logs -->
<!-- /testrunner logs -->

</div>
<!-- /container -->

<!-- test dashboard counts setting -->
<div id='test-count-setting' class='modal bottom-sheet'> 
<div class='modal-content'> 
<h5>Configure Tests Count Setting</h5> 
<input name='test-count-setting' type='radio' id='parentWithoutNodes' class='with-gap'> 
<label for='parentWithoutNodes'>Parent Tests Only (Does not include child nodes in counts)</label> 
<br> 
<input name='test-count-setting' type='radio' id='parentWithoutNodesAndNodes' class='with-gap'> 
<label for='parentWithoutNodesAndNodes'>Parent Tests Without Child Tests + Child Tests</label> 
<br> 
<input name='test-count-setting' type='radio' id='childNodes' class='with-gap'> 
<label for='childNodes'>Child Tests Only</label> 
</div> 
<div class='modal-footer'> 
<a href='#!' class='modal-action modal-close waves-effect waves-green btn'>Save</a> 
</div> 
</div>
<!-- /test dashboard counts setting -->

<!-- filter for step status -->
<div id='step-status-filter' class='modal bottom-sheet'> 
<div class='modal-content'> 
<h5>Select status</h5> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-pass'> 
<label for='step-dashboard-filter-pass'>Pass</label> 
<br> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-fail'> 
<label for='step-dashboard-filter-fail'>Fail</label> 
<br> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-fatal'> 
<label for='step-dashboard-filter-fatal'>Fatal</label> 
<br> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-error'> 
<label for='step-dashboard-filter-error'>Error</label> 
<br> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-warning'> 
<label for='step-dashboard-filter-warning'>Warning</label> 
<br> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-skip'> 
<label for='step-dashboard-filter-skip'>Skipped</label> 
<br> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-info'> 
<label for='step-dashboard-filter-info'>Info</label> 
<br> 
<input checked class='filled-in' type='checkbox' id='step-dashboard-filter-unknown'> 
<label for='step-dashboard-filter-unknown'>Unknown</label> 
</div>
<div class='modal-footer'> 
<a href='#!' class='modal-action modal-close waves-effect waves-green btn'>Save</a> 
</div> 
</div>
<!-- /filter for step status -->

<script src='https://cdn.rawgit.com/anshooarora/extentreports/3caea19ade2c7b195f5ab8aecb6e22541651f5d1/cdn/extent.js' type='text/javascript'></script>

<script>$(document).ready(function() { $('.logo span').html('ExtentReports'); });</script>
<script>


$(document).ready(function() {

});


</script>
</body>
</html>
